Command/RotateStoreCommand.php

```diff
@@ -0,0 +1,87 @@
+<?php
+/**
+ * Created by PhpStorm.
+ * User: jderay
+ * Date: 8/3/15
+ * Time: 6:10 PM
+ */
+
+namespace Giftcards\Encryption\Command;
+
+use Giftcards\Encryption\CipherText\Group;
+use Giftcards\Encryption\CipherText\Rotator\Bounds;
+use Giftcards\Encryption\CipherText\Rotator\ConsoleOutputObserver;
+use Giftcards\Encryption\CipherText\Rotator\Rotator;
+use Giftcards\Encryption\CipherText\Rotator\StoreRegistry;
+use Giftcards\Encryption\Encryptor;
+use Symfony\Component\Console\Command\Command;
+use Symfony\Component\Console\Input\InputArgument;
+use Symfony\Component\Console\Input\InputInterface;
+use Symfony\Component\Console\Input\InputOption;
+use Symfony\Component\Console\Output\OutputInterface;
+
+class RotateStoreCommand extends Command
+{
+    private $rotator;
+
+    public function __construct(
+        StoreRegistry $storeRegistry,
+        Encryptor $cipherTextGenerator
+    ) {
+        $this->rotator = new Rotator($cipherTextGenerator, $storeRegistry);
```

Comment:
i think you should just make the contructor argument to the command the rotator. Law of demeter. Knowledge of how to construct a rotator shouldnt be necessary for the command